.github/workflows/ci.yml

```diff
@@ -30,7 +30,8 @@ on:
       - main
   pull_request:
     paths:
-      - object_store/**
+      - src/**
```

Comment:
we can probably simplify this and not even have a path filter. But this is good too